packages/query/src/block-processor.ts

```diff
@@ -251,6 +252,26 @@ export class BlockProcessor implements BlockProcessorInterface {
       let metadata: Metadata | undefined;
       metadata = await this.querier.shieldedPool.assetMetadata(assetId);
 
+      // If the metadata is for a delegation token, customize its symbol.
+      if (metadata && assetPatterns.delegationTokenPattern.test(metadata.display)) {
+        // We can't trust the validator's self-described name, so use their validator ID.
+        // We know it's delegation_penumbravalid1... so use substrings:
+
+        // TODO: what's the best way to handle delegation tokens to unknown validators?
```

Comment:
If we have made it within this if clause, this means they aren't in our registry. So this will always be unknown validators.